text/0005-coretime-interface.md

```diff
@@ -0,0 +1,176 @@
+# RFC-5: Coretime Interface
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 06 July 2023                                                                                |
+| **Description** | Interface for manipulating the usage of cores on the Polkadot Ubiquitous Computer. |
+| **Authors**     | Gavin Wood                                                                                  |
+
+
+## Summary
+
+In the Agile Coretime model of the Polkadot Ubiquitous Computer, as proposed in RFC-1 and RFC-3, it is necessary for the allocating parachain (envisioned to be one or more pallets on a specialised Brokerage System Chain) to communicate the core assignments to the Relay-chain, which is responsible for ensuring those assignments are properly enacted.
+
+This is a proposal for the interface which will exist around the Relay-chain in order to communicate this information and instructions.
+
+## Motivation
+
+The overall motivation for splitting out functions from the Relay-chain onto System parachains is well understood. An well-understood interface is necessary for ensuring multiple chains are able to coordinate their efforts.
+
+## Requirements
```

Comment:
style: with the template in #2 , this would be a sub-section of "Motivation", as in the motivation for this RFC is to solve a problem with these requirements.